OmniKit/MessageBlocks/ConfigureAlertsCommand.swift

```diff
@@ -108,21 +123,27 @@ public struct ConfigureAlertsCommand : MessageBlock {
             self.alertType = alertType
             
             self.audible = encodedData[0] & 0b1000 != 0
-
+            
             self.autoOffModifier = encodedData[0] & 0b10 != 0
-
+            
             self.duration = TimeInterval(minutes: Double((Int(encodedData[0] & 0b1) << 8) + Int(encodedData[1])))
-
+            
             let yyyy = (Int(encodedData[2]) << 8) + (Int(encodedData[3])) & 0x3fff
-
+            
             if encodedData[0] & 0b100 != 0 {
                 let volume = Double(yyyy * 2) * podPulseSize
                 self.expirationType = .reservoir(volume: volume)
             } else {
                 self.expirationType = .time(TimeInterval(minutes: Double(yyyy)))
             }
+            let beepTypeBits = encodedData[4]
+            guard let beepType = BeepType(rawValue: beepTypeBits) else {
+                throw MessageError.unknownValue(value: beepTypeBits, typeDescription: "beepType")
```

Comment:
Small suggestion: typeDescription is intended to have name of the enum type that we're missing a value for, so `beepType` should be capitalized.